pyyoutube/client.py

```diff
@@ -144,54 +143,28 @@ def _from_client_secrets_file(self, client_secret_path: str):
         Raises:
             PyYouTubeException: missing required key, client_secret file not in 'web' format.
         """
-        try:
-            secrets_data = None
-
-            with open(client_secret_path, "r") as f:
-                secrets_data = json.load(f)
-
-            # For now only 'web' client_secret files are support,
-            # some 'installed' type files can have missing 'client_secret' key
-            if "web" in secrets_data:
-                secrets_data = secrets_data["web"]
 
-                self.client_id = secrets_data["client_id"]
-                self.client_secret = secrets_data["client_secret"]
+        with open(client_secret_path, "r") as f:
+            secrets_data = json.load(f)
 
-                # Set default redirect to first defined in client_secrets file
-                if (
-                    "redirect_uris" in secrets_data
-                    and len(secrets_data["redirect_uris"]) > 0
-                ):
-                    self.DEFAULT_REDIRECT_URI = secrets_data["redirect_uris"][0]
-
-                return
-
-            else:
-                raise PyYouTubeException(
-                    ErrorMessage(
-                        status_code=ErrorCode.INVALID_PARAMS,
-                        message="Only 'web' client_secret file are supported.",
-                    )
-                )
-
-        except KeyError as ke:
+        # For now only 'web' client_secret files are support,
+        # some 'installed' type files can have missing 'client_secret' key
+        if "web" not in secrets_data:
```

Comment:
To support 'installed' type secrets we could do : 

``` Python
secrets_type = list(secrets_data.keys())[0] # get fist key name
if secrets_type not in ["web", "installed"]:
    raise ...

```

Reply:
I don't this use `keys()[0]` is a good idea. Maybe we can do this, just check the key,.

```
credentials = None
for secrets_type in ["web", "installed"]:
    if secrets_type in secrets_data:
        credentials = secrets_data[secrets_type]
if not credentials:
    raise PyYouTubeException(xxxx)
# Then to check client id and client secret
```

Reply:
Good idea